lib/Db/LoginAddressAggregatedMapper.php

```diff
@@ -109,6 +109,24 @@ public function findHistoricAndRecentIpv4(int $threshold, int $maxAge = 0): arra
 		];
 	}
 
+	public function hasSufficientIpV6Data(int $start) {
+		$qb = $this->db->getQueryBuilder();
+
+		$query = $qb
+			->select($qb->createFunction('COUNT(*)'))
+			->from($this->getTableName())
+			->where($qb->expr()->andX(
+				$qb->expr()->notLike('ip', $qb->createNamedParameter('_%._%._%._%')),
```

Comment:
not for now. But would it not make more sense for a future PR to just set a flag in the table if it is v4 or v6?

Reply:
Yeah, that would speed up the queries a bit